ddtrace/llmobs/_integrations/langchain.py

```diff
@@ -136,6 +142,20 @@ def _llmobs_set_meta_tags_from_chat_model(
                     )
         span.set_tag_str(OUTPUT_MESSAGES, json.dumps(output_messages))
 
+    def _llmobs_set_meta_tags_from_chain(
+        self,
+        span: Span,
+        inputs: Union[str, Dict[str, Any], List[Union[str, Dict[str, Any]]]],
+        outputs: Any,
+        error: bool = False,
+    ) -> None:
+        span.set_tag_str(SPAN_KIND, "workflow")
+
+        if inputs is not None:
+            span.set_tag_str(INPUT_VALUE, str(inputs))
+        if not error and outputs is not None:
+            span.set_tag_str(OUTPUT_VALUE, str(outputs))
```

Comment:
We'll need to add a placeholder empty output value for erroneous workflow spans (similar to how we do this for llm/chat_models)
```suggestion
        if error:
            span.set_tag_str(OUTPUT_VALUE, "")
        elif outputs is not None:
            span.set_tag_str(OUTPUT_VALUE, str(outputs))
```